specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,392 @@
+        De verblijfplaats van de persoon kan een ligplaats, een standplaats of een verblijfsobject zijn.
+      example: "0226010000038820"
+    GemeenteVanInschrijving:
+      description: |
+        Een code die aangeeft in welke gemeente de persoon woont, of de laatste gemeente waar de persoon heeft gewoond, of de gemeente waar de persoon voor het eerst is ingeschreven.
+      type: string
+      pattern: ^[0-9]{4}$
+      example: "0518"
+    Huisletter:
+      description: |
+        Een toevoeging aan een huisnummer in de vorm van een letter die door de gemeente aan een adresseerbaar object is gegeven.
+      type: string
+      pattern: ^[a-zA-Z]{1}$
+      example: "a"
+    Huisnummer:
+      description: |
+        Een nummer dat door de gemeente aan een adresseerbaar object is gegeven.
+      type: integer
+      minimum: 1
+      maximum: 99999
+      example: 14
+    Huisnummertoevoeging:
+      description: |
+        Een toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter die door de gemeente aan een adresseerbaar object is gegeven.
+      type: string
+      pattern: ^[a-zA-Z0-9]{1,4}$
+      example: "bis"
+    Locatiebeschrijving:
+      type: string
+      description: |
+        Omschrijving van de ligging van een verblijfsobject, standplaats of ligplaats.
+      example: "Naast de derde brug"
+    NaamOpenbareRuimte:
+      type: string
+    NummeraanduidingIdentificatie:
+      description: |
+        Unieke identificatie van een nummeraanduiding (en het bijbehorende adres) in de BAG.
+      type: string
+      pattern: ^[0-9]{16}$
+      example: "0518200000366054"
+    Postcode:
+      description: |
+        De door PostNL vastgestelde code die bij een bepaalde combinatie van een straatnaam en een huisnummer hoort.
+      type: string
+      pattern: ^[1-9]{1}[0-9]{3}[ ]?[A-Za-z]{2}$
+      example: "2341SX"
+    Straat:
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Tulpstraat"
+    VertrokkenOnbekendWaarheen:
+      type: boolean
+      description: |
+        Indicatie dat de persoon is vertrokken naar het buitenland, maar dat niet bekend is waar naar toe.
+    Woonplaats:
+      description: |
+        Een woonplaats is een gedeelte van het grondgebied van de gemeente met een naam.
+      title: woonplaats naam
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Duiven"
+    AbstractVerblijfplaatsBeperkt:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaVerblijfplaats: '#/components/schemas/GbaVerblijfplaatsBeperkt'
+          VerblijfplaatsBuitenland: '#/components/schemas/VerblijfplaatsBuitenlandBeperkt'
+          Verblijfplaats: '#/components/schemas/VerblijfplaatsBeperkt'
```

Comment:
Verblijfplaats is de generieke naam van het gegeven, dus ook verblijf buitenland, locatie en verblijf onbekend zijn verblijfplaatsen in dit domein. Deze variant is het "Adres".

Reply:
aangepast